src/central.js

```diff
@@ -58,6 +58,22 @@ export function geometricMean(arr) {
   return nthRoot(prod, arr.length);
 }
 
+/**
+ * Return the root mean square of a numeric data array.
+ * The root mean square is the square root of the
+ * mean square (the arithmetic mean of the squares of a set of numbers).
+ * @param {Number[]} arr the data array
+ * @returns {Number} the arithmetic mean of the data array
+ */
+export function rootMeanSquare(arr) {
+  if (min(arr) === undefined) return undefined;
+  let arrSquareSum = 0;
+  for (let i = 0; i < arr.length; i += 1) {
+    arrSquareSum += arr[i] ** 2;
+  }
+  return Math.sqrt((arrSquareSum / arr.length));
```

Comment:
Although this approach solves the problem, it does not make use of previously defined methods in the library. We could use the literal definition of the rms: 
* Compute the mean of the squared values of the elements of the array;
* Compute its square root if it exists.

This would give something like this:

```js
function rms(arr) {
  const xBar = mean(arr.map(elt => elt ** 2));
  return xBar === undefined ? undefined : Math.sqrt(xBar);
}
```